docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md

```diff
@@ -1 +1,290 @@
+```
+php -S localhost:8000 -t . router.php
+```
+Logs will be sent to stderr.
+
+### 1.5.1 Modify Apache Configuration
+
+Modify the Apache configuration file for our development environment.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
+```
+
+**a.** Find & modify the `ServerName` to:
+```
+ServerName localhost
+```
+
+**b.** Find the lines for `DocumentRoot` and `Directory` in Apache and change them to:
+```
+DocumentRoot "/Users/<your_user>/Development/GitHub/Loris/htdocs/"
+<Directory "/Users/<your_user>/Development/GitHub/Loris/htdocs/">
+```
+
+**c.** In the same `<Directory>` block, modify `AllowOverride` to allow all:
+```
+# AllowOverride controls what directives may be placed in .htaccess files.
+# It can be "All", "None", or any combination of the keywords:
+#   AllowOverride FileInfo AuthConfig Limit
+#
+AllowOverride All
+```
+
+**d.** Enable `mod_rewrite` which is commented out by default:
+```
+LoadModule rewrite_module lib/httpd/modules/mod_rewrite.so
+```
+
+**e.** Modify the `User` & `Group` (replace your_user again with your macOS user) to:
+```
+User your_user
+Group staff
+```
+
+#### Restart Apache to load the updated settings
+```
+sudo apachectl -k restart
+```
+
+## 1.6 Install PHP and pkg-config
+
+Type these commands into your terminal and hit enter:
+```
+brew install php72
+brew link php72
+brew services start php72
+brew install pkg-config
+```
+
+### 1.6.1 Modify Apache Configuration for PHP
+
+Modify the Apache configuration file to enable php.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
+```
+
+**a.** Add the line below to `LoadModules` (Change `php@<version>` to the correct version):
+```
+LoadModule php7_module /usr/local/opt/php@<version>/lib/httpd/modules/libphp7.so
+```
+
+**b.** Modify `<IfModule dir_module>` to this:
+```
+<IfModule dir_module>
+    DirectoryIndex index.php index.html
+</IfModule>
+```
+
+**c.** Add the lines below under `</IfModule>`:
+```
+<FilesMatch \.php$>
+    SetHandler application/x-httpd-php
+</FilesMatch>
+```
+
+#### Restart Apache will load the updated settings
+```
+sudo apachectl -k restart
+```
+
+## 1.7 (Optional) Installing Xdebug
+
+Xdebug allows debugging with breakpoints (such as when using an IDE ex. Intellij).
+
+Type into your terminal and hit enter:
+```
+pecl install xdebug
+```
+
+### 1.7.1 Configure Xdebug for local development
+
+**a.** Comment out the line `zend_extensions=“xdebug.so”` from the top of your `php.ini` file (Note: replace `your_php_version` with the correct php version installed):
+```
+sudo open -e /usr/local/etc/php/<your_php_version>/php.ini
+```
+
+b. Create the file `ext-xdebug.ini` with this command:
+```
+sudo touch /usr/local/etc/php/<your_php_version>/conf.d/ext-xdebug.ini
+```
+
+c. Type the following into the `ext-xdebug.ini` file you just created (Note: replace the `zend_extension` path with the correct one on your machine):
+```
+[xdebug]
+zend_extension="/usr/local/Cellar/php@7.2/7.2.14/pecl/20170718/xdebug.so"
```

Comment:
```suggestion
zend_extension="/usr/local/Cellar/php@7.2/7.2.14/pecl/20170718/xdebug.so"
```

It might be good to replace this version info with `<your_php_version>` or something similar.